.github/workflows/build-and-publish.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       max-parallel: 10
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
```suggestion
        os: [ubuntu-latest]
```

Reply:
@leecalcote so no Darwin builds for nighthawk?

Reply:
@kumarabd, How much time do they take to build? Will this org run out of workflow time?